### DISCUSSION TITLE
FF個數請益

### DISCUSSION THREAD

**Discussion by nthuyouwei** [bol-edu/HLS-SOC-Discussions#97] [Q&A]
我想請問fir 這個 FF個數有限制嗎?沒有的話，我也想知道FF個數大概落在哪裡較為正常。

**Comment by Yanlin900414** [comment]
我大概消耗了113個左右
感覺可以比對一下，再來做調整

**Reply by nthuyouwei** [reply]
你加法器是只用一個在全部的設計，還是shift時也有用，其實我不太確定shift可不可以用加法器來做。

**Reply by ghost** [reply]
好像有規定shift必須用bram完成，FF個數沒說所以應該還好吧

**Reply by Yanlin900414** [reply]
我等明天去詢問老師後再做相對應更改，我目前主要FIR計算上使用一個Adder，但在其它設計也有用到加法器。

**Reply by nthuyouwei** [reply]
好不然我原本想說沒用+，我用if條件賦值，跟<<做起來也超級怪!
